bridge/tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "declaration": false,
     "downlevelIteration": true,
     "experimentalDecorators": true,
-    "module": "esnext",
+    "module": "es2020",
```

Comment:
Not sure what this does, can you please explain?

Reply:
https://angular.io/guide/migration-update-module-and-target-compiler-options